README.md

```diff
@@ -52,42 +52,49 @@ echo "PHPVERSION=7.0" > ~/etc/phpversion
 killall php-cgi
 ```
 
+Make sure that your php-cli is set to verion 7.0 with `php -v`.
+If not try to renew your ssh session.
+
 
 ## 4. Install composer
 
 ```sh
 # https://getcomposer.org/download/
 php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
-php -r "if (hash_file('SHA384', 'composer-setup.php') === '070854512ef404f16bac87071a6db9fd9721da1684cd4589b1196c3faf71b9a2682e2311b36a5079825e155ac7ce150d') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
+# If u see "Installer corrupt" it could be that composer-setup is changed.
```

Comment:
# If you see "Installer corrupt" the `composer-setup.php` file may have been updated.